src/helpers/timeHelpers.js

```diff
@@ -43,4 +58,37 @@ export const intervalsMs = {
 	},
 }
 
+export const fillEmptyTime = (prevAggr, timeframe) => {
+	const time = {
+		interval: intervalsMs.lastHour,
+		step: { ammount: 1, unit: 'minute' },
+	}
+	switch (timeframe) {
+		case 'hour':
+			time.interval = intervalsMs.lastHour
+			time.step = { ammount: 1, unit: 'minute' }
+			break
+		case 'day':
+			time.interval = intervalsMs.last24Hours
+			time.step = { ammount: 1, unit: 'hour' }
+			break
+		case 'week':
+			time.interval = intervalsMs.last7Days
+			time.step = { ammount: 6, unit: 'hour' }
+			break
+		default:
+			return prevAggr
+	}
+	const newAggr = []
+	for (
+		var m = moment(time.interval.start);
+		m.diff(time.interval.end, time.step.unit) <= 0;
+		m.add(time.step.ammount, time.step.unit)
+	) {
+		newAggr.push({ value: '0', time: m.unix() * 1000 })
+	}
+	const result = newAggr.map((item, i) => Object.assign({}, item, prevAggr[i]))
```

Comment:
can't use the spread operator rather than `Object.assign` ?

Reply:
yes it would be better